x-pack/plugins/security/server/prompt_page.tsx

```diff
@@ -92,16 +95,28 @@ export function PromptPage({
   return (
     <html lang={i18n.getLocale()}>
       <head>
-        <title>Elastic</title>
+        <title>{customBranding.pageTitle ? customBranding.pageTitle : 'Elastic'}</title>
```

Comment:
__question:__ didn't you mention that product is OK with just `Elastic`?

Reply:
Sorry, I meant for logged in / logged out messages. This one we should still change.

Reply:
Ah, sorry, it's good then :+1: